### PR TITLE
{devel}[GCCcore/9.3.0] libunwind v1.3.1, glog v0.4.0, gflags v2.2.2

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bear-pg0306u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/093d76635a4f27f7a6bf11e6e5a89912 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f245015ad877f1e7372d4a48ac9fd620 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/996245a050d87de56f166b8b05a31e1b for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
bb2c1a09.bb2.cluster - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 0 @ 2.20GHz, Python 2.7.5
See https://gist.github.com/4ff31934e3569be3cb18113c3e2c5049 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2409.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/16f040d619846261b92d09383deeecdd for a full test report.

File: easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+builddependencies = [('binutils', '2.34')]
+
+dependencies = [
+    ('XZ', '5.2.4'),
+]
+
+preconfigopts = 'export LIBS="$LIBS -llzma" && export CFLAGS="$CFLAGS -fuse-ld=bfd" && '
+
+sanity_check_paths = {
+    'files': ['include/libunwind.h', ('lib/libunwind.%s' % SHLIB_EXT, 'lib64/libunwind.%s' % SHLIB_EXT)],
```
Comment (boegel) [inline easybuild/easyconfigs/l/libunwind/libunwind-1.3.1-GCCcore-9.3.0.eb]: @branfosj Just `'lib/libunwind.%s' % SHLIB_EXT` is fine these days, the `lib64` path is considered automatically if `lib/` isn't there

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso-4 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/d2ed46a46ac7f9622935f22d83bb04c3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
generoso-3 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/d9fce54bd8f247982c98775dffe77469 for a full test report.

**Pull request merged**